docs/main_text.md

```diff
@@ -56,9 +62,13 @@ Values are stripped of white space and forced to lowercase.
 | `compound` | `compound`, `compound1`, `compound2`, `compound name`, `drug`, `drugs`, `immunosuppressive drugs` |
 | `time` | `initial time point`, `start time`, `stop time`, `time point`, `sampling time point`, `sampling time`, `time post infection` |
 | `age` | `age`, `patient age`, `age of patient`, `age (years)`, `age at diagnosis`, `age at diagnosis years`, `characteristic [age]`, `characteristics [age]` |
-| `cell_line` | `cell line`, `sample strain` |
+| `cell_line` | `cell line` |
+
+When multiple source keys that map to the same harmonized key are present in metadata from sources, we concatenate the values, separated by `;`.
+For example, a sample with `cell type: B cell` and `tissue: kidney` would become `specimen_part: B cell;kidney` when harmonized.
```

Comment:
Minor question: do we do any sorting on these or is it haphazard (order of the source keys). I might suggest we sort alphabetically, but I can see arguments against, and it isn't particularly important. 

Either way, we should probably make a note about the order of the concatenated values, if only to say it is not meaningful.

Reply:
Tagging @davidsmejia to weigh in on how concatenation is implemented. I will update the text to reflect the answer.

Reply:
We don't sort the values that get harmonized but the order of the keys that we check is consistent. That order is strongly correlated to the order of what is in the table but that looks a bit dated. I can provide an exhaustive list of the checked source fields for each harmonized value if we want to better represent that here.

Reply:
When I say "dated" I actually mean that the list of source values for example `time` should also contain`time`. This is not true for every harmonized key. Also that the way we create variants means that we should either list all possible way `characteristic` is checked or omit it and say that we check a variety of fields that would have that substring in them. ie. `characteristic [ <attribute> ]` and `characteristic_<attribute>` etc

Reply:
Would it be difficult to sort by alphabetical order then? Because we could say that the values are in the order that they were checked and update the table here, but if we could say we sorted it alphabetically that's much easier to explain (and maintain re: the table ordering).



> Also that the way we create variants means that we should either list all possible way `characteristic` is checked or omit it and say that we check a variety of fields that would have that substring in them. ie. `characteristic [ <attribute> ]` and `characteristic_<attribute>` etc

I don't think listing all possible ways is going to be easy to maintain, so let's go the other way.

Reply:
Can you both take a look at https://github.com/AlexsLemonade/refinebio-docs/pull/172/commits/c641186bceef0cf318551fcc8a5ca90da544f31a, @davidsmejia and @jashapiro?

Reply:
> Would it be difficult to sort by alphabetical order then? Because we could say that the values are in the order that they were checked and update the table here, but if we could say we sorted it alphabetically that's much easier to explain (and maintain re: the table ordering).

Yeah that makes sense to me. I will update the implementation to sort the harmonized values. They will be in alphanumeric ascending order. So 0-9 and a-z.